examples/tlsclient.nim

```diff
@@ -0,0 +1,35 @@
+import pkg/[chronos, 
+            chronos/streams/tlsstream,
+            chronicles,     
+            stew/byteutils]  
+
+import ../src/ws
+ 
+proc main() {.async.} =
+    let ws = await webSocketTLSConnect(
+        "127.0.0.1",
+        Port(8888),
+        path = "/wss",
+        protocols = @["myfancyprotocol"],
+        flags = {NoVerifyHost,NoVerifyServerName})
```

Comment:
This flags must not be defaults, because its insecure.

Reply:
That's just an example implementation, it should be fine here.

Reply:
Exactly you should not make examples with such defaults, because people will use your examples in their code.

Reply:
Would this be possible in a test setup running against localhost/127.0.0.1? We need to make sure that this run locally, with as little setup as possible.